engine/robot_logic/traversal.py

```diff
@@ -50,6 +50,54 @@ def traverse_standard(robot_state, unvisited_waypoints, allowed_dist_error, data
 
     return robot_state, unvisited_waypoints
 
+def simple_move_to_target_node(robot_state, target, allowed_dist_error, database):
+    """
+    Moves robot to target + or - allowed_dist_error
+    Arguments:
+        target: target coordinates in the form (latitude, longitude)
+        allowed_dist_error: the maximum distance in meters that the robot 
+        can be from a node for the robot to have "visited" that node
+        database: 
+    """
+    import matplotlib.pyplot as plt
+
+
+    # location error (in meters)
+    distance_away = calculate_dist(target, robot_state.state)
+    robot_state.goal_location = target
+    
+    while (distance_away > allowed_dist_error) and not (robot_state.phase == Phase.AVOID_OBSTACLE):
+        # Error in terms of latitude and longitude, NOT meters
+        x_coords_error = target[0] - robot_state.state[0]
+        y_coords_error = target[1] - robot_state.state[1]
+
+        desired_angle = math.atan2(
+            target[1] - robot_state.state[1], target[0] - robot_state.state[0])
+
+        x_vel = x_coords_error
+        y_vel = y_coords_error
+
+        cmd_v, cmd_w = feedback_lin(
+            robot_state.state, x_vel, y_vel, robot_state.epsilon)
+
+        # clamping of velocities:
+        (limited_cmd_v, limited_cmd_w) = limit_cmds(
+            cmd_v, cmd_w, robot_state.max_velocity, robot_state.radius)
+
+        robot_state.linear_v = limited_cmd_v[0]
+        robot_state.angular_v = limited_cmd_w[0]
+        travel(robot_state, robot_state.time_step *
+               limited_cmd_v[0], robot_state.time_step * limited_cmd_w[0])
```

Comment:
is this based on the new move_to_target_node logic or the old one?

Reply:
This code was based off of the original move_to_target_node

Reply:
uhhh, i dont know if we want to use the old logic, but ig we cant use the new logic too, since it relies on turn_to_target_node; I'm actually not sure but it might be a good idea to ask Shreya about this

Reply:
Todo: keep as original logic. when we test it out, and doesnt meet the intedned tracjeoty, then add back alan's pid fix.

Reply:
Added this todo.